### ISSUE TITLE
Error after updating to v1.10.4

### ISSUE THREAD

**Issue by FaridAghili** [laravel/pint#195]
### Pint Version

1.10.4

### PHP Version

8.2.8

### Description

I
I'm on Windows 11 and here is the error:

```
PHP Fatal error:  Uncaught TypeError: trim(): Argument #1 ($string) must be of type string, bool given in phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/src/Terminal.php:60
Stack trace:
#0 phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/src/Terminal.php(60): trim()
#1 phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/src/Terminal.php(31): HumbugBox436\KevinGH\RequirementChecker\Terminal::initDimensions()
#2 phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/src/Printer.php(28): HumbugBox436\KevinGH\RequirementChecker\Terminal->getWidth()
#3 phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/src/Checker.php(17): HumbugBox436\KevinGH\RequirementChecker\Printer->__construct()
#4 phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/bin/check-requirements.php(22): HumbugBox436\KevinGH\RequirementChecker\Checker::checkRequirements()
#5 C:\Users\farid\AppData\Roaming\Composer\vendor\laravel\pint\builds\pint(12): require('...')
#6 C:\Users\farid\AppData\Roaming\Composer\vendor\bin\pint(119): include('...')
#7 {main}
  thrown in phar://C:/Users/farid/AppData/Roaming/Composer/vendor/laravel/pint/builds/pint/.box/src/Terminal.php on line 60
```

### Steps To Reproduce

Just run Pint on a fresh Laravel project.

**Comment by logue** [comment]
I've getting same error in Laravel v10.15.0 (PHP v8.2.8).

**Comment by romaldyminaya** [comment]
I have just the same error:

![image](https://github.com/laravel/pint/assets/2809147/35cc6b3d-bf95-460b-9c0d-4b863906ecb1)

**Comment by inckmo** [comment]
**Same Error after update Pint**
---
Laravel Pint 1.10.4 on PHP 8.2.8

---
![image](https://github.com/laravel/pint/assets/19841184/ee7c30ff-c045-4ba8-8f96-f7fb013c4c82)

**Comment by MGeurts** [comment]
Same error here on Laravel Pint 1.10.4 on PHP 8.2.8.

**Comment by driesvints** [comment]
Hi all. We're waiting until https://github.com/box-project/box/pull/990 gets merged and tagged. Can't really do anything before that :/

**Comment by wahyupratama18** [comment]
I'm encountering that too today. Maybe you can downgrade it to 1.10.3 instead

**Comment by StephaneKuma** [comment]
Same error here on Laravel Pint 1.10.4 on PHP 8.1.10. I've also tried to downgrade it but the problem was not solved.

**Comment by inckmo** [comment]
> Same error here on Laravel Pint 1.10.4 on PHP 8.1.10. I've also tried to downgrade it but the problem was not solved.

Downgrade works for me.

Make sure you completely remove laravel pint 
using "composer remove laravel/pint --dev" from composer

 then install --- "composer require laravel/pint:1.10.3 --dev"

**Comment by kmaphane** [comment]
> composer require laravel/pint:1.10.3 --dev

Thanks

**Comment by wilderTorres18** [comment]
I have just the same error: 
the solution:
Make sure you completely remove laravel pint
using "composer remove laravel/pint --dev" from composer

then install --- "composer require laravel/pint:1.10.3 --dev"

**Comment by MohammedElattar** [comment]
> 

Thanks

**Comment by nunomaduro** [comment]
Fixed on Pint v1.10.5.
